open-hackathon-server/src/hackathon/__init__.py

```diff
@@ -174,6 +174,7 @@ def init_components():
     factory.provide("admin_manager", AdminManager)
     factory.provide("team_manager", TeamManager)
     factory.provide("guacamole", GuacamoleInfo)
+    factory.provide("host_server_manager", DockerHostManager)
```

Comment:
Looks like `DockerHostManager` already provided in factory with key `docker_host_manager`